lib/modifiers.js

```diff
@@ -42,10 +45,13 @@ function nestedRedact (store, o, path, ns, censor, isCensorFct) {
   const target = get(o, path)
   if (target == null) return
   const keys = Object.keys(target)
-  const length = keys.length
-  for (var i = 0; i < length; i++) {
+  const keysLength = keys.length
+  const pathLength = path.length
+  const pathWithKey = path.concat('')
```

Comment:
this allocation is not needed if isCensorFn is false, we can save some memory here as well.